docs/concepts/client/transient-errors.md

```diff
@@ -0,0 +1,102 @@
+(transient-erros)=
+
+# Transient Errors
+
+Most of the transient errors can be attributed to network issues between the client and the target server or between the
+dependencies of the server like a database. The errors can either be:
+
+1. ignored if an operation produced by a generator or sequence of operations isn't relevant to the overall success.
+2. retried up to a certain limit which assumes that the recovery logic kicks in to repair transient errors.
+3. accept that the operation cannot be successfully completed.
+
+The following sections describe the various features that can be used to handle one or more of the above listed
+scenarios.
+
+## Transient fault handling with retries
+
+The {meth}`~jina.clients.mixin.PostMixin.post` method accepts `max_attempts`, `initial_backoff`, `max_backoff`
+and `backoff_multiplier` parameters to control the capacity to retry requests, when a transient connectivity error
+occurs, using an exponential backoff strategy.
+This can help to overcome transient network connectivity issues which are broadly captured by the
+{class}`~grpc.aio.AioRpcError`, {class}`~asyncio.CancelledError` and the {class}`~jina.excepts.InternalNetworkError`
+exception types.
+
+The `max_attempts` parameter determines the number of sending attempts, including the original request.
+The `initial_backoff`, `max_backoff`, and `backoff_multiplier` parameters determine the randomized delay in seconds
+before retry attempts.
+
+The initial retry attempt will occur at `initial_backoff`. In general, the *n-th* attempt will occur
+at `random(0, min(initial_backoff*backoff_multiplier**(n-1), max_backoff))`.
+
+### Handling gRPC retries for streaming and unary RPC methods
+
+The {meth}`~jina.clients.mixin.PostMixin.post` method supports the `stream` boolean parameter. If set to `True`,
+the **gRPC** server side streaming RPC method will be invoked. If set to `False`, the server side unary RPC method will
+be invoked. Some important implication of
+using retires with **gRPC** are as follows:
+
+1. The built-in **gRPC** retries are limited in scope and are implemented to work under certain circumstances. More
+   details are specified in the [design document](https://github.com/grpc/proposal/blob/master/A6-client-retries.md).
+2. If `stream` parameter is set to True (defaults to True), the retry must
+   be handled as below because the result must be consumed to check for errors in the stream of responses.
```

Comment:
Just use bullets unless we're calling back to specific list items later in the doc